README.md

```diff
@@ -17,30 +17,21 @@ packages using `dnf`, bringing the experience that
 
 ## Installation
 
-Clone this repository (or download `dnfy` directly)
-``` sh
-git clone https://github.com/arctize/dnfy.git
-cd dnfy
+Download `dnfy`:
+
+```sh
+curl -L -o dnfy https://raw.githubusercontent.com/arctize/dnfy/main/dnfy
 chmod +x dnfy # Make sure it's executable
 ```
 
-Inside the directory where the file resides, it can be run directly with
-`./dnfy`.
-
-To make it runnable from anywhere, it needs to be in the `PATH`. Detailed
-information on this can be found
-[here](https://www.baeldung.com/linux/path-variable>).
-
-A quick example:
+Move it to a location in PATH
```

Comment:
I think we could have PATH still pointing to the URL explaining how it works.

Reply:
Sure, makes sense. I will simply the installation process even more, while i'm at it.